setup.py

```diff
@@ -35,6 +35,14 @@
     keywords="databricks hive dbapi",
     packages=["databricks_dbapi"],
     install_requires=["pyhive[hive]"],
+    extras_require={
+        "sqlalchemy": ["sqlalchemy"],
+    },
     include_package_data=False,
     zip_safe=False,
+    entry_points={
+        "sqlalchemy.dialects": [
+            "hive.databricks = databricks_dbapi.sqlalchemy_databricks:DatabricksDialect",
+        ],
+    }
```

Comment:
What's this for?

Reply:
Assuming this is from below to allow for `create_engine("hive+databricks://user:pw@host...")

https://bitbucket.org/zzzeek/sqlalchemy/src/44a9820b4e02f65b3884fa2c016efce9663e4910/README.dialects.rst?at=master

Reply:
what do you think of registering this as just `databricks` but also `databricks.hive` instead? seems a bit more consistent with other dialect registrations

Reply:
PyHive simply opts for hive and presto. See https://github.com/dropbox/PyHive/blob/6925cd74721cc474a02a6759490b2b4c8bef3831/setup.py#L67. I chose `hive+databricks` since it is still essentially the HiveDialect with the databricks dbapi used as the driver. SQL Alchemy defines that part of the url as `dialect+driver` here https://docs.sqlalchemy.org/en/13/core/engines.html. I'm open to doing it the other way as well. Either way, we may want to strongly consider changing the `name` and `driver` attributes in the `DatabricksDialect` class to reflect whichever way we go.